packages/office-ui-fabric-react/src/components/Dropdown/examples/Dropdown.Basic.Example.tsx

```diff
@@ -42,6 +43,7 @@ export class DropdownBasicExample extends React.Component<any, any> {
         />
 
         <Dropdown
+          className="ms-Dropdown-example"
```

Comment:
Yeah, I wanted to put margin back in the example, but make sure it was obviously coming from example code. I can clean it up here.